tests.py

```diff
@@ -0,0 +1,125 @@
+import unittest
+import timesync
+import urllib2
+import httplib
+import mock
+import requests
+import json
+import errno
+
+class TestTimeSync(unittest.TestCase):
+
+    def test_send_time_valid(self):
+        """Tests TimeSync.send_time with valid data"""
+        # Parameters to be sent to TimeSync
+        params = {
+            "duration": 12,
+            "project": "ganet_web_manager",
```

Comment:
splling! also, in the ``timesync-node`` docs and source we generally use ``words-separated-by-hyphens``, so perhaps do the same here.